scripts/install-chef-suse.sh

```diff
@@ -130,7 +131,7 @@ pipe_show_and_log () {
         rm -f $t
         dialog --clear >&3
     fi
-    tee -a /dev/fd/3 /dev/fd/4 > /dev/null
+    tee -a /dev/fd/4 >&3
```

Comment:
> Did you check that this didn't break the script when installing without calling from crowbar? Also, can you reproduce the issue when manually calling the script with the same options as crowbar use?

no not yet, i was working on the backup/restore mostly but i can hopefully do that tomorrow.